app/components/contents/certificate.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Document,
+  Image,
+  Page,
+  StyleSheet,
+  Text,
+  View,
+} from "@react-pdf/renderer"
+
+const styles = StyleSheet.create({
+  page: {
+    flexDirection: "row",
+    backgroundColor: "white",
+    position: "relative",
+  },
+  container: {
+    width: "100vw",
+    height: "100vh",
+  },
+  backgroundImage: {
+    height: "100vh",
+    width: "70vw",
+    position: "absolute",
+    objectFit: "contain",
+    objectPosition: "top right",
+    opacity: 0.05,
+    top: 0,
+    right: 0,
+    zIndex: 1,
+  },
+  section: {
+    margin: "20 30",
+    padding: 20,
+    flexGrow: 1,
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+    zIndex: 10,
+  },
+  title: {
+    fontSize: "30px",
+  },
+  bandungDevIcon: {
+    width: "200px",
+    height: "50px",
+    objectPosition: "center left",
+    objectFit: "contain",
+    marginBottom: "10px",
+  },
+  containerContent: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "20px",
+  },
+  name: {
+    fontSize: "36px",
+    color: "#3C83F6",
+  },
+  containerContentDetail: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "5px",
+  },
+  containerContentFooter: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "flex-end",
+  },
+  signature: {
+    height: "60px",
+    width: "60px",
+  },
+  signatureTitle: {
+    marginTop: "5px",
+    fontSize: "10px",
+  },
+  containerVerificationCertificate: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-end",
+    fontSize: "14px",
+    gap: "4px",
+  },
+  url: {
+    fontSize: "10px",
+  },
+})
+
+interface CertificateType {
+  eventName: string
+  fullName: string
+  date: string
+  url: string
+}
+
+export function Certificate({
+  eventName,
+  fullName,
+  date,
+  url,
+}: CertificateType) {
+  return (
+    <Document>
+      <Page size="A4" style={styles.page} orientation="landscape">
+        <View style={styles.container}>
+          <Image
```

Comment:
The image seems to be disappear, any idea why it happen?

Reply:
I’m not sure why, but I'll check it.

Reply:
https://github.com/diegomura/react-pdf/issues/1736
something to do with CORS,

an alternative way is to change it to base64 string,
I noticed that I encounter the same problem before, fixed it by rendering the image outside the pdf frame and convert it to  base64 string
![image](https://github.com/user-attachments/assets/560aae5d-4ef7-4791-84fb-11e432c661cc)

Feel free to change the implementation if you found a simpler way.

Reply:
I fixed this by adding some attribute in the `src` field object like this :
https://github.com/diegomura/react-pdf/issues/1736#issuecomment-1065524509

Commit : 
https://github.com/bandungdevcom/bandungdev.com/commit/0f68ce83ee16bcd412193b68b3fc774fb4f79a50

Preview : https://bandungdev-pi0eln727-bandungdev.vercel.app/events/certificate/2024-02-18-agustchannel.pdf

Reply:
I have fixed the image issue. I think the problem wasn't due to CORS, but because I tried to access the direct path to 'public/images/...'. So, I changed the source to '{APP_URL}/images/...'. For the signature, I moved it to the environment variables, and I uploaded the image to Uploadcare because I think it involves confidentials. 

Please review it again, @abui-am @agus-wesly @mhaidarhanif .